packages/block-editor/src/components/iframe/style.scss

```diff
@@ -9,9 +9,10 @@
 }
 
 .block-editor-iframe__scale-container.is-zoomed-out {
-	$container-width: var(--wp-block-editor-iframe-zoom-out-container-width, 100vw);
```

Comment:
We should be able to remove this from the JS too since it's no longer used outside the iframe.